googleapis/google-cloud-node
# migrate code from googleapis/nodejs-eventarc

- feat: initial stub of library
- feat!: initial generation of library (#1)
- docs: remove unused comments (#3)
- chore: release 1.0.0 (#2)
- chore(deps): update dependency sinon to v11 (#6)
- fix(deps): google-gax v2.17.1 (#7)
- chore: release 1.0.1 (#8)
- fix: Updating WORKSPACE files to use the newest version of the Typescript generator. (#9)
- chore: release 1.0.2 (#10)
- chore(nodejs): update client ref docs link in metadata (#15)
- fix(deps): google-gax v2.24.1 (#18)
- chore: release 1.0.3 (#19)
- feat: turns on self-signed JWT feature flag (#20)
- chore: release 1.1.0 (#21)
- chore: disable renovate dependency dashboard (#1194) (#22)
- chore: release 1.1.1 (#23)
- chore: relocate owl bot post processor (#26)
- docs(samples): add auto-generated Node samples (#28)
- fix: flag module as GA (#25)
- chore: release 1.1.2 (#29)
- build(node): update deps used during postprocessing (#1243) (#32)
- chore(deps): update dependency @types/node to v16 (#33)
- chore(deps): update dependency sinon to v12 (#34)
- docs(samples): add example tags to generated samples (#35)
- build: add generated samples to .eslintignore (#36)
- meta: do not publish eventarc publishing proto (should be its own library) (#37)
- docs(node): support "stable"/"preview" release level (#1312) (#40)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#41)
- chore: add api_shortname and library_type to repo metadata (#39)
- test(nodejs): remove 15 add 16 (#1322) (#43)
- chore: add codeowners (#47)
- build: update copyright year to 2022 (#45)
- chore: update v2.12.0 gapic-generator-typescript (#48)
- chore(deps): update dependency sinon to v13 (#50)
- docs(nodejs): version support policy edits (#1346) (#53)
- feat: Add Channel and ChannelConnection resources (#51)
- chore(main): release 1.2.0 (#54)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#55)
- chore: update v2.14.2 gapic-generator-typescript (#59)
- build(node): update client library version in samples metadata (#1356) (#66)
- chore(deps): update dependency @types/mocha to v9 (#67)
- chore: remove unused imports (#69)
- chore(deps): update dependency sinon to v14 (#71)
- build!: update library to use Node 12 (#73)
- feat: Add Provider resources (#75)
- chore(main): release 2.0.0 (#74)
- chore(deps): update dependency jsdoc-fresh to v2 (#77)
- chore(deps): update dependency jsdoc-region-tag to v2 (#78)
- feat: support regapic LRO (#79)
- chore(main): release 2.1.0 (#80)
- chore(deps): update dependency linkinator to v4 (#84)
- fix: better support for fallback mode (#87)
- fix: change import long to require (#89)
- fix: do not import the whole google-gax from proto JS (#1553) (#93)
- fix: allow passing gax instance to client constructor (#94)
- chore(main): release 2.1.1 (#90)
- fix: preserve default values in x-goog-request-params header (#97)
- test: use fully qualified request type name in tests (#100)
- fix(deps): use google-gax v3.5.2 (#104)
- chore(deps): update dependency @types/node to v18 (#102)
- chore(deps): update dependency jsdoc to v4 (#106)
- fix: regenerated protos JS and TS definitions (#107)
- feat: Add CMEK support to Eventarc client library (#108)
- chore(main): release 2.2.0 (#98)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
